client/scripts/views/pages/profile/index.ts

```diff
@@ -96,6 +96,7 @@ interface IProfilePageState {
   comments: OffchainComment<any>[];
   loaded: boolean;
   loading: boolean;
+  refresh: boolean;
```

Comment:
It is not clear what refresh does - can we use a more descriptive name to make the code self-documenting?